bolt/include/bolt/Rewrite/RewriteInstance.h

```diff
@@ -490,6 +490,9 @@ class RewriteInstance {
   /// Store all non-zero symbols in this map for a quick address lookup.
   std::map<uint64_t, llvm::object::SymbolRef> FileSymRefs;
 
+  /// FILE symbols.
+  std::vector<object::DataRefImpl> FileSymbols;
```

Comment:
This comment is perfunctory. You might want to consider making it more explicit or just remove it altogether.